CMakeLists.txt

```diff
@@ -65,6 +65,9 @@ if(glew-cmake_BUILD_STATIC)
 	set_target_properties(libglew_static PROPERTIES
 		OUTPUT_NAME "glew"
 		DEBUG_POSTFIX d)
+	target_include_directories(libglew_static PUBLIC
+		"$<BUILD_INTERFACE:${GLEW_SOURCE_DIR}/include>"
```

Comment:
`GLEW_SOURCE_DIR` is not defined. Is it correct? It seems like you want using the `INCLUDE_DIR`, already set on line number 72.

Reply:
`${PROJECT}_SOURCE_DIR` is defined by cmake automatically via the `project` command.

But I'm not sure why I wasn't picking up line 72. I'll look into this. I'm using `glew` as a git submodule and that directory definitely wasn't being picked up.